ld.so.conf

```diff
@@ -1,3 +1,6 @@
+# libc default configuration
+/usr/local/lib
```

Comment:
@sgerrand Hey, there. It's not needed indeed (it might be needed for some software maybe, I don't now really). Though it's been added already, anyways it's both okay to keep or ignore `/usr/local/lib` line.